governance/pyth_staking_sdk/src/constants.ts

```diff
@@ -0,0 +1,18 @@
+import { PublicKey } from "@solana/web3.js";
+
+export const EPOCH_DURATION = 60n * 60n * 24n * 7n; // 1 week
+
+export const POSITION_BUFFER_SIZE = 200;
+export const POSITIONS_ACCOUNT_SIZE = 8 + 32;
```

Comment:
8 is account discriminator size and 32 is the account data size

Reply:
right -- I'm just saying it would be more obvious to name them explicitly:

```ts
const ACCOUNT_DISCRIMINATOR = 8;
const ACCOUNT_DATA_SIZE = 32;
const POSITIONS_ACCOUNT_SIZE = ACCOUNT_DISCRIMINATOR + ACCOUNT_DATA_SIZE;
```

Reply:
Fixed.